dpypelines/pipeline/shared/transforms/sdmx/v20.py

```diff
@@ -0,0 +1,66 @@
+# TODO - these functions need creating properly
+# these are just placeholder functions doing very basic things
+# so that they can be picked up by other scripts
+# delete both stub_ functions and change the other functions as required
+# the output paths from smdx_default_v1() were just made up so change as required too
+
+
+import json
+from pathlib import Path
+
+import pandas as pd
+
+from dpypelines.pipeline.shared.transforms.sdmx.compact.v20.prototype.v1 import (
+    generate_versions_metadata,
+    xmlToCsvSDMX2_0,
+)
+
+
+def sdmx_compact_2_0_prototype_1(input_file: Path):
+
+    csv_out = Path("data.csv")
+    metadata_out = Path("metadata.json")
+
+    xmlToCsvSDMX2_0(input_file, csv_out)
+    generate_versions_metadata(csv_out, metadata_out)
+
+    return csv_out, metadata_out
+
+
+def stub_smdx_default_v1():
```

Comment:
we've defined theses stubbed functions on both `./v20.py` and `./v21.py`. I would move the stub functions to say `./stub.py` and import them in v20 and v21, otherwise we're repeating ourselves.